### PR TITLE
feat: support npm v9

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR adds support for npm v9 and updates CI to use it.

It also updates the PR template to remove the `Before/After` section, which is only really needed for a webapp.

### Context of Change

npm v9 is the latest version of npm.

### Type of Change

- [x] New feature (non-breaking change which adds functionality)

## Test Plan

Tests pass locally. CI passes.

### CODE REVIEW

**Review by justinr1234 — APPROVED** [review]

File: packages/xrpl/HISTORY.md [hunk 1]
```diff
@@ -4,14 +4,15 @@ Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xr
...
-* Null and undefined values in transactions are now treated as though the field was not passed in.
+* Adds support for npm v9
 
 ### Fixed
 * Fixed `ServerState.transitions` typing, it is now a string instead of a number. (Only used in return from `server_state` request)
 * Added `destination_amount` to `PathOption` which is returned as part of a `path_find` request
 * Removed the `decode(encode(tx)) == tx` check from the wallet signing process
 * Fixed the location of `signer_lists` in the `account_info` response so that it matches rippled
+* Guard check for signing algorithm used in `Wallet.generate()`
+* Null and undefined values in transactions are now treated as though the field was not passed in.
```
Comment (JST5000) [inline packages/xrpl/HISTORY.md:15]: Why are these added to this PR?

**Review by JST5000 — APPROVED** [review]
LGTM
